contract/src/lib.rs

```diff
@@ -63,7 +63,7 @@ impl Default for BattleshipContract {
 pub fn verify_receipt(str: &String, method_id: &MethodID) -> Vec<u32> {
     let as_bytes = base64::decode(str).unwrap();
     let receipt = bincode::deserialize::<Receipt>(&as_bytes).unwrap();
-    receipt.verify(&method_id);
+    if receipt.verify(&method_id).is_err() { panic!("receipt did not verify") }
```

Comment:
if you want a message, you can use `.expect("some message")`, but to have the same behavior as before, we could just do `.unwrap()`

Reply:
.unwrap() is a great idea - done